build.sh

```diff
@@ -10,27 +10,6 @@ TOOLS_DIR=$SCRIPT_DIR/tools
 NUGET_EXE=$TOOLS_DIR/nuget.exe
 CAKE_EXE=$TOOLS_DIR/Cake/Cake.exe
 
-# Define default arguments.
-SCRIPT="build.cake"
```

Comment:
Hmm, I thought that the cake.exe defaulted as well. If I run
```
$ mono tools/Cake/Cake.exe
```
it defaults to using build.cake


Are these defaults not the same as the ones in cake.exe?